gm17/Cargo.toml

```diff
@@ -22,19 +22,21 @@ edition = "2018"
 ################################# Dependencies ################################
 
 [dependencies]
-algebra = { path = "../algebra", features = [ "parallel" ] }
-ff-fft = { path = "../ff-fft" }
-r1cs-core = { path = "../r1cs-core" }
+algebra = { path = "../algebra", default-features = false }
 bench-utils = { path = "../bench-utils" }
-
-rand = { version = "0.7" }
-rayon = { version = "1" }
-smallvec = "1.1.0"
+ff-fft = { path = "../ff-fft", default-features = false }
+r1cs-core = { path = "../r1cs-core", default-features = false }
+rand = { version = "0.7", default-features = false }
+rayon = { version = "1", optional = true }
+smallvec = "1.2"
 
 [dev-dependencies]
 csv = { version = "1" }
 
 [features]
+default = ["parallel"]
+std = ["algebra/std", "ff-fft/std", "r1cs-core/std"]
+parallel = ["std", "algebra/parallel", "ff-fft/parallel", "rayon"]
 print-trace = [ "bench-utils/print-trace" ]
 
 [[example]]
```

Comment:
I think you have to add `required-features = ["std"]` like [here](https://github.com/scipr-lab/zexe/blob/ac165844ab6fa6fc7fef88cf3f6a60e810d97fb3/groth16/Cargo.toml#L46)